src/main/java/io/vertx/openapi/contract/OpenAPIVersion.java

```diff
@@ -21,13 +21,7 @@
 import io.vertx.core.Future;
 import io.vertx.core.Vertx;
 import io.vertx.core.json.JsonObject;
-import io.vertx.json.schema.Draft;
-import io.vertx.json.schema.JsonFormatValidator;
-import io.vertx.json.schema.JsonSchema;
-import io.vertx.json.schema.JsonSchemaOptions;
-import io.vertx.json.schema.JsonSchemaValidationException;
-import io.vertx.json.schema.OutputUnit;
-import io.vertx.json.schema.SchemaRepository;
+import io.vertx.json.schema.*;
```

Comment:
no star imports

https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcards-for-class